db/views/patient_summaries_v01.sql

```diff
@@ -0,0 +1,88 @@
+SELECT
+p.recorded_at,
+CONCAT(date_part('year', p.recorded_at), ' Q', EXTRACT(QUARTER FROM p.recorded_at)) AS registration_quarter,
```

Comment:
All date calculations in the SQL-land will be in UTC. So we shouldn't directly use these dates as is in the views anywhere.